TribalWars/src/Infrastructure/Mapping/ModelToResourceProfile.cs

```diff
@@ -0,0 +1,23 @@
+﻿using ApplicationCore.Entities;
+using AutoMapper;
+using Infrastructure.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Mapping
+{
+    public class ModelToResourceProfile : Profile
```

Comment:
Class name is suggesting we can find here Model to Resource mapping profile, but in fact it's a Entity to ViewModel profile.